db/migrate/20240402194159_create_version_metadatas.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateVersionMetadatas < ActiveRecord::Migration[7.0]
+  def change
+    create_table :version_metadata, primary_key: %i[druid version] do |t|
+      t.string      :druid, null: false, index: true
```

Comment:
In practice I think we only ever care about druid/version pairs, so you could consider dropping `index: true` here (and from the db schema)?

Reply:
ok